<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Координаты</string>
    </property>
    <widget class="QLabel" name="label_X_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Xн = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_start_Edit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y_start">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Yн = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_start_Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y_end">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Yк = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_end_Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_end_Edit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_X_end">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Xк = </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Метод</string>
    </property>
    <widget class="QComboBox" name="method_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="draw_Btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_Btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>500</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет отрезка</string>
    </property>
    <widget class="QComboBox" name="penColor_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>330</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет фона</string>
    </property>
    <widget class="QComboBox" name="backgroundColor_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
